pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -187,10 +190,74 @@ func NewSQLTupleIterator(rows *sql.Rows) *SQLTupleIterator {
 		rows:     rows,
 		resultCh: make(chan *storage.TupleRecord, 1),
 		errCh:    make(chan error, 1),
+		firstRow: nil,
```

Comment:
```suggestion
		firstRow: nil, // GUARDED_BY(mu)
```

It doesn't do anything, but i read that it's used in other languages https://clang.llvm.org/docs/ThreadSafetyAnalysis.html. It only helps us developers see where the mutex is being used.